pkg/crc/ssh/ssh.go

```diff
@@ -14,16 +14,16 @@ import (
 )
 
 type Runner struct {
-	ip            string
-	port          int
-	privateSSHKey string
+	ip             string
+	port           int
+	privateSSHKeys []string
 }
 
-func CreateRunnerWithPrivateKey(ip string, port int, privateKey string) *Runner {
+func CreateRunnerWithPrivateKey(ip string, port int, privateKeys ...string) *Runner {
```

Comment:
The method name should also be `CreateRunnerWithPrivateKeys`, or just `CreateRunner` since there is no longer a method with that name.

Reply:
done 👍